dev-util/ebuildtester/Manifest

```diff
@@ -1 +1,2 @@
 DIST ebuildtester-42.tar.gz 22852 BLAKE2B 0e672c70ed854b3ee459010590da22811ed06e0397d45e51b8ad1a86047aa3fbd61321a7a9bdf1b583c3a66246030bb30dbef4dc8acc1e10bc3b141095e08ff4 SHA512 c2fe0dbca673c51d686cc19954928efb3ab1e82c4d8b8548d272f8fc60a5fffd1898fe33dc26feebc5bc73a71207f5f1615e18036b264e76a09b9dc6d94168fa
+DIST ebuildtester-44.tar.gz 23982 BLAKE2B 96ebdaa1278068d83805b3d9b86b0191ded2028dc042c3ed5dbd22cee38831f07f8a69a3e9ba955d189b64b20b17669f2587ab259cf4f80b490a055bbbf98053 SHA512 a391821999fe3b559dddb6bda326e25d8b64e59330b87fe7f0453aefcf4145720d72bde562d791c147b7a9c3901a676a7184db3b8ce1c8b5f8923ca3473b09ef
```

Comment:
You're removing the old version but not removing the old version from Manifest?

Reply:
As there was two revisions for the same version, even when I removed one revision the other revision still downloads the same tar file and hence it is still present in the manifest file.

Reply:
Yes, sorry for being dumb. :o